src/HTMLGenerator/testSuite/sequencer.py

```diff
@@ -3,18 +3,52 @@
 import pathlib
 import errno
 import json
+import time
 from pathlib import Path
 from asserter import TestCase
 from tqdm import tqdm
 import utils
+import subprocess
 from utils import *
 import sys
 sys.path.append("..")
 import converter
-from converter import convert
+from converter import run_converter
 
+CURRENT_CONFIGURATION_PATH = "currentConfiguration.txt"
+NOTIFICATION_PATH = "complete.txt"
+LABVIEW_PATH = Path("C:/Program Files/National Instruments/LabVIEW 2017/LabVIEW.exe")
 ERROR_REPORT_JSON = r'errorReport.json'
 
+def convert_test_suite(top_level_output_directory, top_level_input_directory, json_with_filepaths, path_to_current_working_directory = Path.cwd()):
+    current_configuration_absolute_path = Path(path_to_current_working_directory / CURRENT_CONFIGURATION_PATH)
+    notification_absolute_path = Path(path_to_current_working_directory / NOTIFICATION_PATH)
+
+    try:
+        if os.path.isfile(notification_absolute_path):
+            os.remove(notification_absolute_path)                
+    except:
+        pass
```

Comment:
Isn't this already being done in run_converter()?

Reply:
removed